migrations/anser.go

```diff
@@ -67,12 +68,18 @@ func (opts Options) Application(env anser.Environment) (*anser.Application, erro
 		},
 	}
 
+	githubToken, err := evergreen.GetEnvironment().Settings().GetGithubOauthToken()
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
i'd pass evgEnv to Application from the entry point rather than calling GetEnvironment here.

Reply:
addressed everything here, good to go?